cashu/wallet/wallet_deprecated.py

```diff
@@ -261,6 +261,7 @@ async def request_mint_deprecated(self, amount) -> PostMintQuoteResponse:
             paid=False,
             state=MintQuoteState.unpaid.value,
             expiry=decoded_invoice.date + (decoded_invoice.expiry or 0),
+            pubkey=None
```

Comment:
```suggestion
```
Optional parameter doesn't need definition

Reply:
mypy was complaining about this because it's defined as optional but doesn't have a default value.